src/App.tsx

```diff
@@ -1,27 +1,55 @@
 import React from 'react';
-import './App.scss';
-
-const App: React.FC = () => {
-  const timerId: NodeJS.Timer = setInterval(() => {
-    const date: Date = new Date();
-
-    // eslint-disable-next-line
-    console.log(date.toLocaleTimeString());
-  }, 1000);
-
-  // eslint-disable-next-line
-  console.log(timerId);
-
-  return (
-    <div className="App">
-      <h1>React clock</h1>
-      <p>
-        Current time:
-        {' '}
-        {/* Print the time here instead of DevTools */}
-      </p>
-    </div>
-  );
+import { Clock } from './components/Clock';
+
+type Props = {};
+
+type State = {
+  name: number,
+  visible: boolean,
 };
 
+class App extends React.Component<Props, State> {
+  state = {
+    name: 0,
+    visible: true,
```

Comment:
потрібно явно прописувати імена властивостей для розуміння(якщо що то так самі ментори вимагають))))
```suggestion
    visibleOfClock: true,
```